app/router.js

```diff
@@ -3,7 +3,19 @@ import config from './config/environment';
 
 const Router = Ember.Router.extend({
     location: config.locationType,
-    rootURL: config.rootURL
+    rootURL: config.rootURL,
+    willTransition(oldInfos, newInfos) {
+        const disableList = ['settings', 'moderation'];
+        const prevRoute = oldInfos.length ? oldInfos[oldInfos.length - 1].name.split('.') : null;
```

Comment:
It'd be nicer to use `Ember.Router._routePath(infos)` (which I discovered looking at the [parent `willTransition`](https://github.com/emberjs/ember.js/blob/v2.14.1/packages/ember-routing/lib/system/router.js#L331) -- this feels like a case where using a private method is better than doing it manually), and have the list be `['preprints.provider.moderation', 'preprint.provider.settings']`